### ISSUE TITLE
Licenses of training material

### ISSUE THREAD

**Issue by bkmgit** [ultralytics/ultralytics#24879]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Yolo v26 is trained on Coco and Objects 365 datasets.  The objects 365 dataset does not seem to have licensing information for the annotations, see https://github.com/sshao0516/Objects365/issues/20

Coco dataset seems to have licensing information for annotations and each image:
https://github.com/cocodataset/cocoapi/issues/551

The contributions of Ultralytics in creating a model are released under AGPL and under a non-AGPL paid license. Do the licenses of the training data also need to be observed when using Yolo v26 both with the open source AGPL model and with the non-AGPL license?

A related question was asked  in https://github.com/ultralytics/ultralytics/issues/8146, but it does not address observation of licenses for the photos, for example Flickr allows the following licenses:

- All Rights Reserved.
- CC BY-SA 4.0 — Attribution-ShareAlike.
- CC BY-ND 4.0 — Attribution-NoDerivs.
- CC BY-NC 4.0 — Attribution-NonCommercial.
- CC BY-NC-SA 4.0 — Attribution-NonCommercial-ShareAlike.
- CC BY-NC-ND 4.0 — Attribution-NonCommercial-NoDerivs.

see:
https://www.flickrhelp.com/hc/en-us/articles/40048459562388-Copyright-Licenses-on-Flickr

Are any of the photos used under one of the above licenses? If so, do these propagate to the Yolo v6 model released under the AGPL?  If one obtains the non-AGPL Yolo v6 license do any of the above licenses not propagate to it should the datasets include photos with these licenses?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @bkmgit, thank you for your interest in Ultralytics 🚀! This is an automated response to help get your issue routed quickly, and an Ultralytics engineer will also assist soon.
We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python) and [CLI](https://docs.ultralytics.com/usage/cli) usage examples and where many of the most common questions may already be answered.
For a licensing and usage question like this, please share any additional context that may help, such as the exact YOLO26 model variant you are referring to and your intended usage scenario, so the team can review it more efficiently 🔍
If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example) to help us debug it.
If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips).
Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.
## Upgrade
Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [**Python>=3.8**](https://www.python.org/) environment with [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:
```bash
pip install -U ultralytics
```
## Environments
YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):
- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo26"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>
## Status
<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>
If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes) and [Tasks](https://docs.ultralytics.com/tasks) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by glenn-jocher** [comment]
Yes — third-party dataset rights are separate from the Ultralytics YOLO license and still need to be followed whether you use AGPL-3.0 or an Enterprise License; our [Ultralytics License](https://www.ultralytics.com/license) covers YOLO code/models, while [Objects365’s own terms](https://www.objects365.org/download.html) state the annotations are `CC BY 4.0` and the images remain subject to Flickr Terms of Use, so an Enterprise License does not waive COCO/Objects365 or upstream image-license obligations, and for a definitive view on whether any source-image terms propagate to trained weights in your use case, please have counsel review the dataset terms.
